package.json

```diff
@@ -84,6 +85,7 @@
     "eslint-plugin-switch-case": "^1.1.2",
     "husky": "^8.0.3",
     "jest": "29.4.3",
+    "jest-when": "^3.5.2",
```

Comment:
[jest-when](https://github.com/timkindberg/jest-when) is MIT-licensed.

It allows us to specify required input arguments when configuring `jest` mocks, e.g.
```javascript
when(jest.fn())
  .calledWith(<required input arguments>)
  .mockReturnValue(<desired output>)
```